### ISSUE TITLE
Term Request: Honeycomb-shaped subepithelial corneal opacities

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11741]
**Preferred term label:**
Honeycomb-shaped subepithelial corneal opacities

**Synonyms**
* Reticular subepithelial corneal opacities
* Honeycomb corneal opacification
* Reticular corneal dystrophy pattern

**Definition (free text, please give PubMed ID)**
Symmetrical, reticular, or honeycomb-patterned subepithelial corneal opacities located at the level of the Bowman layer and superficial stroma, typically sparing the peripheral cornea, caused by fibrocellular pannus with wavy saw-toothed configuration replacing the Bowman layer (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Subepithelial corneal opacities (HP:0008039)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Thiel-Behnke corneal dystrophy (TBCD) (ORPHA:98960, OMIM:602082)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508

---
